projects/baby_driver/scripts/test_spi_exchange.py

```diff
@@ -0,0 +1,141 @@
+"""This Modules tests methods in spi_exchange.py"""
+
+import unittest
+from unittest.mock import patch
+
+from spi_exchange import spi_exchange
+from gpio_port import GpioPort
+from can_util import Message
+from message_defs import BABYDRIVER_DEVICE_ID, BABYDRIVER_CAN_MESSAGE_ID, BabydriverMessageId
+
+OK_STATUS = 0
+FAILING_STATUS = 1
+class TestSPIExchange(unittest.TestCase):
```

Comment:
Nitpick: please put a blank line before the class declaration. I like the named constants instead of just using 0 and 1! Also, usually the recommendation is to treat acronyms like SPI as words in CamelCase: `TestSpiExchange`.